Dell/recovery_backend.py

```diff
@@ -776,6 +776,20 @@ def validate_driver_package(self, package, sender=None, conn=None):
         logging.debug("Validation complete: valid %s" % valid)
         self.report_package_info(valid, description, error_warning)
 
+    #The funcution is used to recovery Dell Hybrid Client
+    @dbus.service.method(DBUS_INTERFACE_NAME,
+        in_signature = 'b', out_signature = '', sender_keyword = 'sender',
+        connection_keyword = 'conn')
+    def enable_boot_to_restore_dhc(self, reboot, sender=None, conn=None):
+        """Enables the default one-time boot option to be recovery"""
+        self._reset_timeout()
```

Comment:
I believe this function is missing a policy kit privilege check like the regular boot to restore.
Something to consider is, how about just making the modification and then calling `enable_boot_to_restore` directly?

Reply:
Different from "enable _boot_to_restore" function,  I need to add create dhc flag part. So I call request_mount function to  mount recovery partition for create dhc file. Meanwhile, have policy kit privilege check(com.dell.recoverymedia.create) in request_mount function, so if add  policy kit privilege check like  "enable _boot_to_restore" function, then select recovery dhc will ask user input password two times.

Reply:
OK make sense.